randovania/games/prime3/logic_database/Bryyo - Reptilicus.txt

```diff
@@ -287,6 +287,8 @@ Extra - asset_id: 13902157786380317936
           All of the following:
               Grapple Swing
               Space Jump Boots or Simple Bomb Jump/Spring Space Jump
+          # https://youtu.be/vYt0nqk75I4
+          Morph Ball Bomb and Morph Ball and Bomb Jump (Intermediate) and Movement (Intermediate)
```

Comment:
I wonder how those compares to other Intermediate Bomb jumps

Reply:
This is the only intermediate bomb jump in the db. And there's only one other advanced, which is used as for a Single room OOB (lol)
Imo, this is pretty straightforward and doesn't have any softlock risk. And fwiw I got it first try after only watching the video once.